src/enclave_proc/commands.rs

```diff
@@ -46,14 +58,42 @@ pub fn run_enclaves(
     .map_err(|e| {
         e.add_subaction("Failed to construct EnclaveManager with given arguments".to_string())
     })?;
+
+    // Launch parallel computing of PCRs
+    let path = args.eif_path.clone();
+    let handle = std::thread::spawn(move || {
+        let mut eif_reader = EifReader::from_eif(path).map_err(|e| {
+            new_nitro_cli_failure!(
+                &format!("Failed initialize EIF reader: {:?}", e),
+                NitroCliErrorEnum::EifParsingError
+            )
+        })?;
+        get_pcrs(
```

Comment:
this should call eif_reader.get_pcrs(). We have no business in going and touching fields that should be encapsulated inside EifReader.

Reply:
I preferred to use an EIF utility function to avoid duplicated code. This function is called by EifReader as well as the EifBuilder as they use the same logic to calculate PCRs.